frontend/workflows/k8s/src/delete-pod.tsx

```diff
@@ -8,40 +8,44 @@ import {
   FeatureOff,
   FeatureOn,
   MetadataTable,
+  NotePanel,
   Resolver,
   SimpleFeatureFlag,
   useWizardContext,
 } from "@clutch-sh/core";
 import { useDataLayout } from "@clutch-sh/data-layout";
-import type { WizardChild } from "@clutch-sh/wizard";
 import { Wizard, WizardStep } from "@clutch-sh/wizard";
 import _ from "lodash";
 
-import type { ConfirmChild, ResolverChild, WorkflowProps } from ".";
+import type { ConfirmChild, ResolverChild, VerifyChild, WorkflowProps } from ".";
 
 const PodIdentifier: React.FC<ResolverChild> = ({ resolverType, notes = [] }) => {
   const { onSubmit } = useWizardContext();
   const resolvedResourceData = useDataLayout("resourceData");
   const resolverInput = useDataLayout("resolverInput");
-
+  const resolverNotes = notes.filter(note => note.location === "resolver");
```

Comment:
This pattern is probably ok for now but I think this comes back to the earlier thread around using context for complex components like the resolver.

However, this will break existing functionality for anyone using this workflow that doesn't specify a location.

Reply:
Do you think it is ok if we show the new notes in both places? Like on the resolver we show both notes, then only the new ones at the next step. I think that could be a good fix and still keep backwards functionality.